### PR TITLE
Add successfully deployed stage

### LINKED ISSUE #2233: JSON output is not including a stage for the endpoints
**Describe the bug**
When the `okteto deploy` command is executed with `json` output, the logs with the endpoints list don't include the stage, so the UI is filter them out and the endpoints are not included in the logs. As you can see here, the stage is empty.

```
{"level":"info","stage":"","message":"Retrieving endpoints...","timestamp":1643990945}
{"level":"info","stage":"","message":" i  Endpoints available:\n  - https://movies-ifbyol.staging/.***.net/\n  - https://movies-ifbyol.staging/.***.net/api\n","timestamp":1643990945}
```

Not sure if this was done in purpose or we should include a stage to be included in the UI. If the stage has to be included, I think a new stage has to be included 

**To Reproduce**
Execute a pipeline from the UI in an Okteto instance with the structured logs. You will see that the endpoints are not printed in the logs

**Expected behavior**
Endpoints should have a stage as the rest of json output

**Desktop (please complete the following information):**

**Additional context**
This is part of structured logs for pipelines

### PR DESCRIPTION
**Pull request by jLopezbarb**
Signed-off-by: Javier López Barba <javier@okteto.com>

Fixes #2233

## Proposed changes
- When all the commands are being executed set the stage as "successfully deployed"

### CODE REVIEW

**Review by pchico83 — CHANGES_REQUESTED** [review]
let's wait for a final decision for https://github.com/okteto/okteto/issues/2233

**Comment by jLopezbarb** [comment]
Closed for now. More information available on #2233

**Pull request closed without merging**
